doc/source/api/ast_class.rst

```diff
@@ -1,7 +1,7 @@
 AST tree
```

Comment:
AST (abstract syntax tree) " as the title. Using "AST tree" is theoretically repeating the word "tree" because tree is already part of the "AST" abbreviation. And using only AST in the title seems like it might be confusing.